src/runtime-rs/crates/agent/src/kata/agent.rs

```diff
@@ -12,7 +12,8 @@ use ttrpc::context as ttrpc_ctx;
 use kata_types::config::Agent as AgentConfig;
 
 use crate::{kata::KataAgent, Agent, AgentManager, HealthService};
-
+use logging::{VMM_DRAGONBALL_LOGGER, AGENT_LOGGER, HYPERVISOR_LOGGER, RESOURCE_LOGGER, RUNTIMES_LOGGER, VIRT_CONTAINER_LOGGER, SERVICE_LOGGER, SHIM_LOGGER};
```

Comment:
Why does this crate need to introduce all the other loggers besides AGENT_LOGGER?